railties/lib/rails/rack/logger.rb

```diff
@@ -7,6 +7,9 @@
 module Rails
   module Rack
     # Sets log tags, logs the request, calls the app, and flushes the logs.
+    #
+    # Log tags can be an Array containing: methods that the <tt>request</tt> object responds to, a Proc
```

Comment:
Just repurposing existing terminology in the docs. *"Log tags"* is more, clear I think. When I hear the word *"taggers"* I think of people who do graffiti or work in the garment industry.